zds/pages/urls.py

```diff
@@ -19,6 +20,7 @@
     path("technologies/", about, name="pages-technologies"),
     path("association/", association, name="pages-association"),
     path("contact/", ContactView.as_view(), name="pages-contact"),
+    path("accessibilite", accessibilite, name="pages-accessibilite"),
```

Comment:
Toutes nos URL doivent se terminer par une barre oblique
```suggestion
    path("accessibilite/", accessibilite, name="pages-accessibilite"),
```